src/style/animatableManager.js

```diff
@@ -0,0 +1,146 @@
+import _ from 'lodash';
+import * as Animatable from 'react-native-animatable';
+
+
+/** Animations Definitions */
+const definitions = {
+  indexEntrance: {
+    from: {opacity: 0, translateY: 40},
+    to: {opacity: 1, translateY: 0},
+  },
+};
+
+class AnimatableManager {
+  constructor() {
+    this.loadCustomDefinitions(definitions);
+    this.animations = getAnimations();
+  }
+
+  loadCustomDefinitions(customDefinitions) {
+    if (customDefinitions) {
+      Animatable.initializeRegistryWithDefinitions(customDefinitions);
+      this.updateDefinitions(customDefinitions);
+    }
+  }
+
+  loadSlideByHeightDefinitions(height, sufix) {
+    const definition = {};
+    // bottom
+    definition[`slideInUp_${sufix}`] = {
+      from: {translateY: height},
+      to: {translateY: 0},
+    };
+    definition[`slideOutDown_${sufix}`] = {
+      from: {translateY: 0},
+      to: {translateY: height},
+    };
+    // top
+    definition[`slideInDown_${sufix}`] = {
+      from: {translateY: -height},
+      to: {translateY: 0},
+    };
+    definition[`slideOutUp_${sufix}`] = {
+      from: {translateY: 0},
+      to: {translateY: -height},
+    };
+    // relative
+    definition[`growUp_${sufix}`] = {
```

Comment:
growUp/Down is not part of slide definitions, maybe we need to accept the animation type also as a param

Reply:
Or we can just call the "slideUp/Down" without the In/Out indication